pin/page_objects/pin_page.py

```diff
@@ -0,0 +1,67 @@
+"""
```

Comment:
the base /pin directory looks like it can be removed entirely, since you have pin logic in /selenium_tests/page_objects/pin_page.py.